lbrynet/daemon/Downloader.py

```diff
@@ -62,9 +61,7 @@ def download_path(self):
         return os.path.join(self.download_directory, self.downloader.file_name)
 
     def _check_status(self, status):
-        stop_condition = (status.num_completed > 0 or
-                status.running_status == ManagedEncryptedFileDownloader.STATUS_STOPPED)
-        if stop_condition and not self.data_downloading_deferred.called:
```

Comment:
This is related to the last unit test you commented out. 

I think the problem here was that if the user manually stops the download before the first blob is finished.  But I think looking a the daemon, that should probably not be possible for them to do now. Was there a reason you changed this?

Reply:
>Was there a reason you changed this?

It caused a false positive in the stop condition

>should probably not be possible for them to do now

It can happen if the daemon gets shut down, which should now be handled by `cancel`